lldb/unittests/Process/gdb-remote/GDBRemoteCommunicationClientTest.cpp

```diff
@@ -593,6 +593,10 @@ TEST_F(GDBRemoteCommunicationClientTest, WriteMemoryTags) {
                  "E03", false);
 }
 
+// Prior to this verison, constructing a std::future for a type without a
+// default constructor is not possible.
+// https://developercommunity.visualstudio.com/t/c-shared-state-futuresstate-default-constructs-the/60897
+#if _MSC_VER >= 1932
```

Comment:
Shouldn't this have a `!defined(_MSC_VER) ||` part at the beginning, in order to retain the test on all non-MSVC configurations?

Reply:
Done.